jax/_src/scipy/signal.py

```diff
@@ -33,6 +33,68 @@
 from jax._src.typing import Array, ArrayLike
 from jax._src.util import canonicalize_axis, tuple_delete, tuple_insert
 
+@_wraps(osp_signal.fftconvolve)
+def fftconvolve(in1: ArrayLike ,in2: ArrayLike, mode: str ="same",axes=None) -> Array:
+  # sanity checks
+  check_arraylike('fftconvolve', in1, in2)
+  in1, in2 = promote_dtypes_inexact(in1, in2)
+  if in1.ndim == in2.ndim == 0:  # scalar inputs
+    return in1 * in2
+  elif in1.ndim != in2.ndim:
+    raise ValueError("in1 and in2 should have the same dimensionality")
+  elif in1.size == 0 or in2.size == 0:  # empty arrays
+    return jnp.array([], dtype=in1.dtype)
+  # warn current limitations
+  if mode != "same":
+    raise NotImplementedError("current fftconvolve() uses only mode=same argument")
+  if axes is not None:
+    raise NotImplementedError("current fftconvolve() does not use axes argument")
+
+  s1 = in1.shape
+  s2 = in2.shape
+
+  axes = range(in1.ndim)
+  # Axes of length 1 can rely on broadcasting rules for multipy,
+  # no fft needed.
+  axes = [a for a in axes if s1[a] != 1 and s2[a] != 1]
+
+  try:
+    jnp.broadcast_shapes(tuple(np.delete(np.array(s1),axes)), 
```

Comment:
I don't think you actually need this check given how `axes` is constructed (all the remaining dimensions will have `1` on either side, so they will always be broadcast-compatible. I think you can safely delete lines 61-65, what do you think?